app/src/main/java/org/jellyfin/mobile/utils/UIExtensions.kt

```diff
@@ -46,6 +48,18 @@ fun View.applyWindowInsetsAsMargins() {
     }
 }
 
+fun View.fadeIn() {
+    alpha = 0f
+    isVisible = true
+    with(animate()) {
+        alpha(1f)
+        @Suppress("MagicNumber")
+        duration = 300L
+        interpolator = LinearOutSlowInInterpolator()
+        start()
+    }
```

Comment:
```suggestion
    animate().apply {
        alpha(1f)
        @Suppress("MagicNumber")
        duration = 300L
        interpolator = LinearOutSlowInInterpolator()
        withLayer()
    }
```
- start() is not needed, will automatically start
- withLayer() enables hardware acceleration

Reply:
- neat, didn't know
- that's not the default??